packages/frames.js/src/middleware/default.ts

```diff
@@ -0,0 +1,11 @@
+import { farcasterHubContext } from "./farcasterHubContext";
+import { framesjsMiddleware } from "./framesjsMiddleware";
+import { renderResponse } from "./renderResponse";
+
+export const defaultMiddleware = [
```

Comment:
we are exporting this twice? Shouldn't we import this in `createFrames.ts` ?

Reply:
my bad, fixing

Reply:
fixed